doc_src/cmds/psub.rst

```diff
@@ -6,7 +6,7 @@ psub - perform process substitution
 Synopsis
 --------
 
-*COMMAND1* ( *COMMAND2* | ``psub`` [**-F** | -**-fifo**] [**-f** | **--file**] [**-s** *SUFFIX*] )
+*COMMAND1* ( *COMMAND2* | ``psub`` [**-F** | -**-fifo**] [**-f** | **--file**] [(**-s** | **--suffix**) *SUFFIX*] )
```

Comment:
Not sure but I guess it's true that humans will parse this correctly in practice. 

> you won't see the parentheses in most manpages.

Isn't this because most traditional commands rarely ever need `|` because they list mutually exclusive options on separate lines?
For example, in GNU coreutils I only saw stty(1) use `|` (inside square brackets); usually they it's something like `ls [OPTION]... [FILE]...`.

We OTOH often write something like `**-s** | **--long-option**`. Unless we change our synopsis style to just `[*OPTIONS*]`, I think we just have different requirements than most traditional programs. So I wonder how comparable man pages solve this.
Git use parens, for example git-merge(1) has

	git merge (--continue | --abort | --quit)

Maybe we should avoid this scenario somehow? git-merge(1) lists only short options in the synopsis, not sure if that would suit us.

Reply:
Yeah most manpages seem to only list short options, but I think long options are much easier to understand. `lsof` uses the no-parentheses style but I have come around to the idea of keeping them having looked at your examples.

Reply:
Yeah I think with parens it's more logical but I can drop these commits if we want to decide later.